spec/lib/pg_query/deparse_spec.rb

```diff
@@ -97,6 +97,11 @@
         it { is_expected.to eq query }
       end
 
+      context 'COUNT DISTINCT' do
+        let(:query) { 'SELECT count(distinct a) FROM x WHERE y IS NOT NULL' }
```

Comment:
I think this needs to be `SELECT count(distinct "a") FROM "x" WHERE "y" IS NOT NULL` (with the added `"`)